content/committees/steering/_index.ja.md

```diff
@@ -12,7 +12,7 @@ date: 2020-01-05T15:50:25-04:00
 * Will Lyons (**委員長**)- Oracle、Ed Bratt - 代理
 * Kenji Kazumura - 富士通、Yuichi Kusano - 代理
 * Ian Robinson - IBM、Alasdair Nottingham - 代理
```

Comment:
This line is incorrect.
Can you update to 
* Alasdair Nottingham - IBM, Emily Jiang - alternate
The following 2 files are correct.

Reply:
I created a patch to resolve this! #1794